tests/test_minimal_requirements.txt

```diff
@@ -8,3 +8,4 @@ pyyaml>=4.2b1
 coverage~=4.5.1
 codecov~=2.0.15
 unyt~=2.8.0
+more_itertools~=8.4
```

Comment:
For minimal requirements do we want to tie to such a close version as regular test_requirements? Is it worth trying a lower version number?

Reply:
Actually this _is_ a minimal requirement: `more_itertools.mark_ends` was introduced in version 8.4.0 (https://more-itertools.readthedocs.io/en/stable/versions.html#id3)